mainmatter.md

```diff
@@ -314,12 +314,17 @@ workshop (Ralf Küsters, Guido Schmitz).
 
 # Security Considerations {#Security}
 
-The prevention of token replay at a different
-endpoint (see (#Objective_Replay_Different_Endpoint)) is achieved through
-the binding of the DPoP proof to a certain URI and HTTP method.
-However, DPoP does not achieve the same level of protection as, for
-example, OAuth Mutual TLS [@I-D.ietf-oauth-mtls], as described in the
-following.
+In DPoP, the prevention of token replay at a different endpoint (see
+(#Objective_Replay_Different_Endpoint)) is achieved through the
+binding of the DPoP proof to a certain URI and HTTP method. DPoP does
+not, however, achieve the same level of protection as TLS-based
+methods such as OAuth Mutual TLS [@I-D.ietf-oauth-mtls] or OAuth Token
+Binding [@I-D.ietf-oauth-token-binding], as described in the
+following. TLS-based mechanisms can leverage a tight integration
```

Comment:
It's a bit unclear "as described in the following" is referring to. The following sentence? The following sub-section (9.1.  DPoP Proof Replay)?  Something else?

Maybe just remove the ", as described in the following"?

Reply:
Thanks. Looks good to me now.